cmd/terramate/cli/cli.go

```diff
@@ -250,46 +285,100 @@ func (c *cli) run() error {
 		return nil
 	}
 
+	logger := log.With().
+		Str("action", "run()").
+		Str("stack", c.wd()).
+		Logger()
+
 	if c.parsedArgs.Changed {
+		logger.Trace().
+			Msg("`Changed` flag was set.")
+
+		logger.Trace().
+			Msg("Create new git wrapper.")
 		git, err := newGit(c.root(), c.inheritEnv, true)
 		if err != nil {
 			return err
 		}
+
+		logger.Trace().
+			Msg("Check git default remote.")
 		if err := c.checkDefaultRemote(git); err != nil {
 			return err
 		}
+
+		logger.Trace().
+			Msg("Check git default branch was updated.")
 		if err := c.checkLocalDefaultIsUpdated(git); err != nil {
 			return err
 		}
 	}
 
+	logger.Debug().
+		Msg("Handle input command.")
 	switch c.ctx.Command() {
 	case "version":
+		logger.Debug().
+			Msg("Get terramate version.")
 		c.log(terramate.Version())
 	case "plan graph":
+		log.Trace().
+			Str("actionContext", "cli()").
+			Str("stack", c.wd()).
+			Msg("Handle `plan graph`.")
```

Comment:
```suggestion
		logger.Trace().Msg("Handle `plan graph`.")
```

Rationale: we already have the logger + no need to break lines if it fits a single line